apps/frontend/src/utils/rgb2hex.ts

```diff
@@ -0,0 +1,6 @@
+export const rgb2hex = (rgb) =>
```

Comment:
You should be able to reuse the kepler function instead :
rgbToHex
- import it from `dist/utils/color-utils.js`
- if required add custom typedef in `apps/shared/models/types/kepler.gl/index.d.ts`